css/style.css

```diff
@@ -67,51 +66,43 @@ a:active {
     text-decoration: underline;
 }
 
-#page-intro{
-  background: url(../images/background.png);
-  background-repeat: repeat repeat; 
-  height: 782px;
-}
-
 .wrapper {
-  max-width: 1280px;
-  margin: auto;
+  max-width: 1024px;
+  margin: 0px auto;
 }
 
-.intro-text{
- padding-top: 25%;
- color: #fff0f0;
- text-align: center;
+#large-header {
+  background: url(../images/background.png);
+  background-repeat: repeat repeat; 
+  height: 100vh;
```

Comment:
up to you -- just be careful with 100vh height headers. you'd be surprised how many people don't attempt to scroll pages that don't indicate there's anything else. there's very few drawbacks to having it be, say, 80vh instead.